### DISCUSSION TITLE
JWT plugin supports /.well-known/jwks.json endpoint

### DISCUSSION THREAD

**Discussion by ozooxo** [Kong/kong#7784] [Ideas and feature requests]
Currently, the JWT plugin supports a [`rsa_public_key` input attribute](https://docs.konghq.com/hub/kong-inc/jwt/#create-a-jwt-credential) for RS256 algorithm. In practice, we'll [need to](https://docs.konghq.com/kubernetes-ingress-controller/1.3.x/guides/using-consumer-credential-resource/) create a Kubernetes opaque secret (with the base64 encoded value of that plain `-----BEGIN PUBLIC KEY-----` file), and link it to the `KongConsumer` CRD. This added a lot of complexity. It will be very nice if JWT plugin can support passing the URL of the /.well-known/jwks.json endpoint, so the system can get the public key by itself.

**Comment by ltartarini** [comment]
Is there any update/plugin related to this topic?
